requirements.txt

```diff
@@ -44,3 +44,6 @@ xlrd==1.2.0; python_version >= '3.8'
 XlsxWriter==1.1.2
 xlwt==1.3.*
 zeep==3.4.0
+pdf2image==1.16.0
+pyzbar==0.1.8
+python3-qrtools==2.1
```

Comment:
I 'm sorry to say that you cannot add those requirements for some reasons:
1) the `l10n_ch` module is really an optional module and we cannot make those requirements mandatory for those who don't use this module.
2) A Debian package **must** exists to install the Python Package on Debian distributions. Also the required version must be the same as the Debian package in current stable release.
I did not find a Debian package for pdf2image and the [python3-qrtools](https://packages.debian.org/bullseye/python3-qrtools) versions does not match
3) As you can see in your [build logs](http://runbot80.odoo.com/runbot/static/build/13377080-master/logs/install_all_split.txt), python3-qrtools==2.1 is not installable.

So, the way to go is to verify in the code that you can import the needed modules and warn the user if it's ot the case. You should have a look how it is done in other l10n_* modules that need some additional python libs.